src/main/java/com/company/M11/Main.java

```diff
@@ -0,0 +1,69 @@
+package com.company.M11;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+//        task1();
+//        task2();
+//        task3();
+//        task4();
+//        task5();
+    }
+
+    private static void task1() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+
+        AtomicInteger index = new AtomicInteger(0);
```

Comment:
why do we need the AtomicInteger type here?

Reply:
fixed